### DISCUSSION TITLE
BTC, Dash and Doge support

### DISCUSSION THREAD

**Discussion by GiMa-Maya** [WalletConnect/walletconnect-monorepo#2621] [Developer Support]
Hey all. 
I’m GiMa from Maya Protocol (fork of THORChain). Maya is a crosschain liquidity protocol that allows swaps between native assets such as BTC, ETH, DASH and more. 

To make it easier for our users to use our crosschain swaps, we would like to see Doge and Dash integrated into Wallet Connect (bitcoin should be similar). 

I’m in a Discord group with Crypblizz and Dash dev and in a Discord group with Crypblizz and a Doge dev. Crypblizz told me he’s more active on here than on Discord, hence this message. 

The devs are ready to work on this. Our question is, what is the best way to start this integration of Dash and Doge into Wallet Connect? They are different than the chains already integrated of course (as UTXO’s) so the regular guide does not apply to us. 

Would love to hear from you!
GiMa 

UPDATE: The Maya and THORChain team can assist with Bitcoin.

**Comment by MuckT** [comment]
Dash, Doge and BTC don't follow the Ethereum Virtual Machine (EVM) standard. You can see a list of supported chains [here](https://docs.walletconnect.com/2.0/advanced/multichain/chain-list).

**Reply by GiMa-Maya** [reply]
I am aware, that’s why we want to add them to WC.

**Reply by BitHighlander** [reply]
Cosmos is not an EVM as well.

**Reply by DimonPanamera** [reply]
I am aware, that’s why we want to add them to WC.

**Comment by GiMa-Maya** [comment]
UPDATE: My team (Maya) and THORChain will help with Bitcoin support.

**Reply by kriblis** [reply]
Is any hope that WalletConnect will support BTC and BCH in the near future?

**Reply by lironSendCrypto** [reply]
Hey is there any update on the matter..?

**Comment by BitHighlander** [comment]
+1 if wallet connect wants to be multi-chain it can't just lock itself to EVM's. I too would be interested in supporting this development and integrating it into KeepKey Desktop.

**Comment by sidvip** [comment]
@GiMa-Maya - were you able to support BTC, Doge, and Lite through wallet connect? Or you are trying to get it manual, will you please tell ?

**Comment by BitHighlander** [comment]
![Screen Shot 2023-10-18 at 2 48 36 PM](https://github.com/WalletConnect/walletconnect-monorepo/assets/4935895/0b0e09ba-f6c8-4cac-b0e2-2a42905cc246)
Its sad, Wallet Connect could be a universally used Wallet protocol, but seeing the lack of interest in supporting more than basic EVM's is disappointing.
